framework/bin/run_mutation.pl

```diff
@@ -266,6 +272,17 @@ sub _run_mutation {
     $project->{prog_root} = "$root";
     $project->checkout_vid($vid) or die "Checkout failed";
 
+    my $num_excluded_mutants = 0;
+    if (defined $EXCL) {
+      system("cp $EXCL $root/") == 0 or die "Cannot copy exclude file";
```

Comment:
Yeah, perhaps we do not need to copy it. We could add another parameter to `Mutation::mutation_analysis(...)` and `Mutation::mutation_analysis_ext(...)` (and therefore to `Project::mutation_analysis(...)` and `Project::mutation_analysis_ext(...)` as well) to get rid of that hard-coded string. About `d4j-mutation`, should we also add a `-e` option to that script?